docs/DeveloperGuide.adoc

```diff
@@ -729,12 +729,69 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |see my application's logo |easily identify the application
+
+|`* * *` |user |add a user with a non-alphabet name |add my friend with a non-alphabet name (e.g. Nguyễn)
+
+|`* * *` |user |use command shortcuts |achieve what I want in a shorter time
+
+|`* * *` |user |add and remove tags |
+
+|`* * *` |user |sort my contacts in A-Z order |see my contacts easier
+
+|`* * *` |user |list contacts with specific tags |
+
+|`* * *` |careless user |see a comfirmation prompt after the 'clear' command |know if I really want to clear my contacts
```

Comment:
The application is supposed to be a single command execute application. Hence having user prompts would be contradictory.

Reply:
Can you kindly point us to the specs that says that this is a single command execute application? I tried to look into the user and dev guide but I can't find it. :)

We thought that this is a rather useful feature, exclusively for the clear command only because it wipes out the entire contact list. Even thought the user can use undo, it is still good to have a confirmation (probably can save some stack space too if it was executed by accident).